scripts/run-dependencies

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+HERE=$(realpath "$(dirname $0)")
+
+## Start database
+"$HERE"/../db/scripts/build
+"$HERE"/../db/scripts/run
+
+# Populate db with dummy data
+docker exec packit-db psql -U packituser -d packit -a -f /packit-dummy/import.sql
```

Comment:
This fails because the db isn't ready, you need to leave [this line](https://github.com/mrc-ide/packit/pull/6/files#diff-4dd3e8495a99556b02e15b8567535ee7ad8ad4ef83868fca5b1a37e5556d481dL11) in place. But this will also fail because you have moved initialising the schema out of `db/scripts/run`, so the `packet` table won't exist when you try to import the dummy data!